src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -11,18 +16,46 @@
  * This class holds all the methods for the intake which collects POWER CELLS
  * into the robot.
  */
-public class Intake extends OverridableSubsystem {
-    private CANSparkMax sparkMax;
+public class Intake extends OverridableSubsystem implements MovableSubsystem {
+    private CANSparkMax cellIntake;
+    private AnalogPotentiometer potentiometer;
+    private WPI_TalonSRX intakeOpener;
 
     public Intake() {
-        sparkMax = new CANSparkMax(robotConstants.can.kIntakeSparkMax, MotorType.kBrushless);
-        sparkMax.setInverted(robotConstants.intakeConstants.kIntakeReversed);
-        sparkMax.setIdleMode(IdleMode.kCoast);
-        sparkMax.burnFlash();
+        cellIntake = new CANSparkMax(robotConstants.can.kIntakeSparkMax, MotorType.kBrushless);
+        cellIntake.setInverted(robotConstants.intakeConstants.kCellIntakeReversed);
+        cellIntake.setIdleMode(IdleMode.kCoast);
+        cellIntake.burnFlash();
+
+        intakeOpener = new WPI_TalonSRX(robotConstants.can.kIntakeTalonSRX);
+        intakeOpener.setInverted(robotConstants.intakeConstants.kIntakeOpenerReversed);
+        intakeOpener.setNeutralMode(NeutralMode.Coast);
+        intakeOpener.configSupplyCurrentLimit(new SupplyCurrentLimitConfiguration(true,
+                robotConstants.intakeConstants.kCurrentLimit, robotConstants.intakeConstants.kThresholdLimit,
+                robotConstants.intakeConstants.kTriggerThresholdTime));
+
+        potentiometer = new AnalogPotentiometer(robotConstants.pwm.kIntakePotentiometer,
+                robotConstants.intakeConstants.kPotentiometerAngleMultiplier,
+                robotConstants.intakeConstants.kPotentiometerOffset);
+    }
+
+    public double getAngle() {
+        return potentiometer.get();
     }
 
+    /**
+     * Overrides the cell collector motor
+     */
     @Override
     public void overriddenMove(double power) {
-        sparkMax.set(power);
+        cellIntake.set(power);
+    }
+
+    /**
+     * Moves the intakeOpener motors
+     */
+    @Override
+    public void move(double power) {
+        intakeOpener.set(power);
     }
```

Comment:
why is one move and the other override??? they should be in functions that the name tells the user what it is